ui/pages/token-allowance/index.scss

```diff
@@ -16,8 +16,8 @@
 
   a.token-allowance-container__verify-link {
     width: fit-content;
-    margin-inline-start: 96px;
-    margin-inline-end: 96px;
+    margin-inline-start: 90px;
+    margin-inline-end: 90px;
```

Comment:
If this is for this link you could just use `margin-inline-start: auto; margin-inline-end: auto;` if you are trying to center the element

Reply:
You are right. Thanks! 
Fixed.